ios/NativeModules/Multipeer.m

```diff
@@ -0,0 +1,88 @@
+#import "Multipeer.h"
+
+@interface Multipeer()
+
+@end
+
+@implementation Multipeer
+
+RCT_EXPORT_MODULE();
+
+RCT_EXPORT_METHOD(init:(NSString *)name address:(NSString *)address avatar:(NSString *)avatar) {
+  [self initialize:name address:address avatar:avatar];
+}
+
+RCT_EXPORT_METHOD(advertiseOn) {
+  [self advertiseToggle:true];
+}
+
+RCT_EXPORT_METHOD(advertiseOff) {
+  [self advertiseToggle:false];
+}
+
+RCT_EXPORT_METHOD(scan) {
+  [self scanStart];
+}
+
+#pragma mark -
+#pragma mark Public Methods
+- (void)initialize:(NSString *)name address:(NSString *)address avatar:(NSString *)avatar {
```

Comment:
coudn't tell the energy hit. but seems like some cool things we could test.

1. Invite by Multipeer. So in a room, just tap their icons and send the invite payload over p2p. More or less we have this already with the shared deeplinks.
2. I think cooler could be to test a 'Create a Local Group' that just opens up the invite to anyone listening, and when they join they also open up the invite, etc etc.